pages/doc/wavefront_internal_metrics.md

```diff
@@ -12,7 +12,10 @@ You can:
 * Clone and modify one of the Operations for Applications Usage integration dashboards.
 * Create your own dashboard, query these metrics in charts, and create alerts for some of these metrics.
 
-{% include note.html content="The internal metrics are [**ephemeral**](metric_types.html#metric-types-per-retention-period) and not convertible to persistent." %}
+Most of the internal metrics are [**ephemeral**](metric_types.html#metric-types-per-retention-period) and not convertible to persistent. Exceptions are the following internal metrics, which are persistent:
```

Comment:
Most of the internal metrics are [**ephemeral**](metric_types.html#metric-types-per-retention-period) and can not be converted to persistent except for the following internal metrics:

Reply:
Hmm, I think this way is a bit vague what the exception is. Sounds to me that they can be converted to persistent but they are persistent.